.upstream-tests/test/std/language.support/support.limits/limits/numeric.limits.members/infinity.pass.cpp

```diff
@@ -53,10 +53,12 @@ int main(int, char**)
     test<__int128_t>(0);
     test<__uint128_t>(0);
 #endif
+#if !defined(_MSC_VER)
```

Comment:
Why are these disabled on MSVC?

Reply:
Complaints of division by zero. I was going to @ you and ask about this one. Should we include host <limits> and use INFINITY?

Reply:
Can we silence the warning with a pragma?

Reply:
This is an outright error:

```
C:\Users\wmaxey\Documents\temp\libcudacxx\.upstream-tests\test\std\language.support\support.limits\limits\numeric.limits.members\infinity.pass.cpp(56): error #39-D: division by zero

C:\Users\wmaxey\Documents\temp\libcudacxx\.upstream-tests\test\std\language.support\support.limits\limits\numeric.limits.members\infinity.pass.cpp(57): error #39-D: division by zero
```

Reply:
Hmm. Okay.

Yeah let's use INFINITY. I'd rather not have this commented out on MSVC.

Reply:
Updated in latest commit and tested on MSVC 14.30/16/12.